lib/src/style/mapbox_styles.dart

```diff
@@ -6,6 +6,9 @@ class MapboxStyles {
   /// Mapbox Standard is a general-purpose style with 3D visualization.
   static const String STANDARD = "mapbox://styles/mapbox/standard";
 
+  /// Mapbox Standard Satellite
+  static const String STANDARD_SATELLITE = "mapbox://styles/mapbox/standard-satellite";
```

Comment:
Should this be "mapbox://styles/mapbox/standard-satellite" or "mapbox://styles/mapbox-map-design/standard-satellite"?

Reply:
@pjleonard37 the URI is correct https://mapbox.slack.com/archives/CE2P14FDM/p1721908730747699?thread_ts=1721322574.128349&cid=CE2P14FDM

Reply:
Sounds good 👍